src/web/demo.rkt

```diff
@@ -271,6 +273,10 @@
     (define post-body (request-post-data/raw req))
     (define post-data (cond (post-body (bytes->jsexpr post-body)) (#t #f)))
     (define resp (with-handlers ([exn:fail? (λ (e) (hash 'error (exn->string e)))]) (fn post-data)))
+    (if (hash-has-key? resp 'error)
+        (eprintf "Error handling request: ~a\n" (hash-ref resp 'error))
+        (eprintf "")
```

Comment:
What's the function of this empty eprintf?

Reply:
Was an empty else statement, I changed it to contain "Success handling request" message.

Reply:
There's a when statement in racket which is an if statement with no else block